code/src/components/Footer.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Footer = () => {
```

Comment:
For some of your JS files (like this one), the indentation is 4 spaces instead of 2, so it makes the code unnecessarily wide (and we should follow the 2 spaces guideline).